### ISSUE TITLE
does this support the new moneroV?

### ISSUE THREAD

**Issue by zifootball** [lucasjones/cpuminer-multi#92]
does this support the new moneroV? monero hard fork

**Comment by mtl1979** [comment]
Nope... Hasn't been updated in about 3 years... Would love to see cpuminer to support CryptoNight, CryptoHeavy and CryptoLight variants... cpuminer-multi works a lot better than xmr-stak or xmrig on low-end machines.

**Comment by SeppPenner** [comment]
No. But you can use https://github.com/hyc/cpuminer-multi/. It works properly, I've tested it with minergate.com on an Ubuntu system.

**Install:**
 
```bash
sudo apt-get install wget
wget https://github.com/hyc/cpuminer-multi/archive/daemon.zip
unzip daemon.zip
cd daemon
./autogen.sh
./configure
make
```

**Run:**

```bash
./minerd --help
```

**Example:**
```bash
./minerd -a cryptonight -o stratum+tcp://xmr.pool.minergate.com:45700 -u my@email.com -p x -t 1
```


Please see also: http://franzhuber23.blogspot.de/2018/04/mining-monero-v7-aka-monerov7-on-ubuntu.html
